types/src/on_chain_config/execution_config.rs

```diff
@@ -255,6 +255,17 @@ impl BlockGasLimitType {
         }
     }
 
+    pub fn use_module_publishing_block_conflict(&self) -> bool {
```

Comment:
can you change default to true in the default_for_genesis function?

Reply:
ok